### ISSUE TITLE
Deprecations in generated Scala classes with Scala 2.13.1

### ISSUE THREAD

**Issue by fbonicel** [julianpeeters/sbt-avrohugger#78]
Hi,

Found some deprecations in the generated scala classes from avro schema 

```
object JavaConverters in package collection is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
[warn]             Seq((scala.collection.JavaConverters...
```
sbt-avrohugger version : 2.0.0-RC22 

build.sbt
```
sourceGenerators in Compile += (avroScalaGenerateSpecific in Compile).taskValue
```

Thanks for this great sbt plugin !

François

**Comment by fsarradin** [comment]
FYI https://github.com/julianpeeters/avrohugger/issues/135

**Comment by julianpeeters** [comment]
thanks all, fixed  in avrohugger 1.0.0-RC25 and sbt-avrohugger 2.0.0-RC25
